apollo-router/src/plugins/csrf/mod.rs

```diff
@@ -27,6 +26,7 @@ pub(crate) struct CSRFConfig {
     /// set unsafe_disabled = true to disable the plugin behavior
     /// Note that setting this to true is deemed unsafe.
     /// See <https://developer.mozilla.org/en-US/docs/Glossary/CSRF>.
+    /// TODO rename this to enabled. This is in line with the other plugins and will be less confusing.
```

Comment:
Should it be done before merging this PR ?

Reply:
No, 2.n or 3.0